test/verify/check-kubernetes

```diff
@@ -26,6 +26,7 @@ import kubelib
 
 @skipImage("No kubernetes packaged", "debian-stable", "debian-testing", "ubuntu-1604", "ubuntu-stable", "fedora-i386")
 @skipImage("No cockpit-kubernetes packaged", "continuous-atomic", "fedora-atomic", "rhel-atomic")
+@skipImage("Irrelevant for rhel-7.5 branch", "rhel-7-5")
```

Comment:
Would this be better if it would just be `@skip("Irrelevant for rhel-7.5 branch")`.  The tests will fail for any image, not just rhel7-5.  Also, on a cursory glance, the two mentions of "rhel 7.5" here might be confusing since they look similar but refer to different things.

Reply:
Ack, will do that. While I'm pushing again, I'll also backport the relevant bits from commit 1bbeb2d608746c53ebb8 to fix semaphore.

Reply:
Pushed.

Reply:
Pushed.